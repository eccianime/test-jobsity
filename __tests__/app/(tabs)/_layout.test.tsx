import TabLayout from "@/app/(tabs)/_layout";
import { render } from "@testing-library/react-native";

jest.mock("expo-router", () => ({
  Tabs: jest.fn(() => null),
  useContextKey: jest.fn(),
}));

jest.mock("@/components/tabs", () => ({
  TabBarIcon: jest.fn(() => null),
  TabBarLabel: jest.fn(() => null),
}));

describe("TabLayout", () => {
  it("should render the initial tab", () => {
    const { toJSON } = render(<TabLayout />);
    expect(toJSON()).toMatchSnapshot();
  });
});
