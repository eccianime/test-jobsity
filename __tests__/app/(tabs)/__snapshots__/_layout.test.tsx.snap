// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabLayout should render the initial tab 1`] = `null`;
